dist/IO/lib/IO/Socket.pm

```diff
@@ -120,7 +120,7 @@ sub connect {
 	if (defined $timeout && ($!{EINPROGRESS} || $!{EWOULDBLOCK})) {
 	    require IO::Select;
 
-	    my $sel = new IO::Select $sock;
+	    my $sel = IO::Select->new( $sock );
```

Comment:
I've not looked but at one point some of the POD examples also suggested indirect notation

Reply:
I looked on metacpan. it looks like it's been corrected at this point so we're all good.